public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -132,6 +131,7 @@ export class ElasticDatasource
     }
     this.languageProvider = new LanguageProvider(this);
     this.timeSrv = getTimeSrv();
+    this.maybeEs7Info = null;
```

Comment:
Why is this maybe?

Reply:
(went with a different approach after discussion)